Utils/Elasticsearch/tools/load_mapping.sh

```diff
@@ -1,20 +1,25 @@
 #!/bin/sh
 usage() {
   cat >&2 <<EOF
-  Usage: $0 [-e <ElasticSearch url>] <mapping_file>
-    -e -- url where we may found elasticsearch. localhost:9200 is default
+  Usage: $0 [-c <ElasticSearch config>] <mapping_file>
+    -c path/to/confg/file. Default is $(dirname $0)/../config/es
 EOF
   exit 1
 }
 
-ES='localhost:9200'
-while getopts e: opt; do
+ES_CONFIG="$(dirname $0)/../config/es"
+ES_HOST='127.0.0.1'
+ES_PORT='9200'
+
+while getopts c: opt; do
   case $opt in		
-    e) ES="$OPTARG";;
+    c) [ -f "$OPTARG" ] && ES_CONFIG="$OPTARG" || echo "Config $OPTARG not found! Will use default values!" ;;
     *) usage;
   esac
 done
+[ -f "$ES_CONFIG" ] && . $ES_CONFIG || echo "Could not found es_confg file $ES_CONFIG! Will use default values" >&2
```

Comment:
The first check (in the `case` clause) might be excessive; also, it tells about error to STDOUT, while the second check -- to STDERR.

Reply:
We should generate warning if user configuration file not exists and we must not tries to load default config if it not exists. 
So we should have two checks.

STD{OUT,ERR} issue fixed.

Reply:
I see, so it is to have possibility to fall back to the default config file even if user has specified its own.
Not sure if I would do same, still it makes sense :) Thank you!